ipfx/stim_features.py

```diff
@@ -18,6 +17,7 @@ def get_stim_characteristics(i, t, test_pulse=True):
     start_idx = di_idx[start_idx_idx] + 1   # shift by one to compensate for diff()
     end_idx = di_idx[-1]
     if start_idx >= end_idx: # sweep has been cut off before stimulus end
+        print("foo", start_idx, end_idx, di_idx)
```

Comment:
should this be more informative than just foo?

Reply:
that is a debug print, good catch :cactus: